### ISSUE TITLE
Add support for streaming requests (attach/exec/port-forward)

### ISSUE THREAD

**Issue by feiskyer** [kubernetes/frakti#41]
Refer https://github.com/kubernetes/kubernetes/issues/29579.

- [x] execsync
- [x] exec
- [x] attach
- [ ] port-forward

This could be done by using [kubelet streaming liberay](https://github.com/kubernetes/kubernetes/tree/master/pkg/kubelet/server/streaming).

**Comment by Crazykev** [comment]
@feiskyer I can pick this up if it's ok with you.

**Comment by feiskyer** [comment]
@Crazykev It includes four interfaces, we can impl them one by one:

- execsync
- exec
- attach
- port-forward

Which one will you start first?

**Comment by Crazykev** [comment]
exec and exec-sync I guess, not very sure about port-forward yet.

**Comment by Crazykev** [comment]
`exec` and `attach` is blocked by lacking of hyper `TTYResize` api, `port-forword` is blocked by lacking of pod level port-forword support in hyper. I'll continue work here once hyperd is ready.

**Comment by feiskyer** [comment]
> exec and attach is blocked by lacking of hyper TTYResize api, port-forword is blocked by lacking of pod level port-forword support in hyper. I'll continue work here once hyperd is ready.

Moved tty resize to #50.

**Comment by feiskyer** [comment]
Port forward is not supported now, we should discuss whether/how to support it. Open another issue #51 for it.
